lib/hawkular/operations/operations_api.rb

```diff
@@ -431,8 +431,14 @@ def prepare_payload_hash(ignored_params, hash)
     end
 
     def to_camel_case(str)
-      ret = str.split('_').collect(&:capitalize).join
-      ret[0, 1].downcase + ret[1..-1]
+      subs = str.split('_')
+      if subs.length > 1
+        ret = subs.collect(&:capitalize).join if subs.length > 1
```

Comment:
Isn't this second `if subs.length > 1` redundant ?

Reply:
whoops, my bad, good eye.